packages/web/src/trpc.ts

```diff
@@ -0,0 +1,5 @@
+import { createTRPCReact } from '@trpc/react-query'
+
+import type { AppRouter } from '../../trpc-server'
```

Comment:
You should be able to do 
```suggestion
import type { AppRouter } from '@audius/trpc-server'
```